webpack.config.js

```diff
@@ -44,7 +44,9 @@ function configure(env, webpackOpts) {
           vendor_A_F: new RegExp('node_modules/[a-fA-F]'),
           vendor_G_O: new RegExp('node_modules/[g-oG-O]'),
           vendor_P_Q: new RegExp('node_modules/[^a-oA-Or-zR-Z]'),
-          vendor_R_Z: new RegExp('node_modules/[r-zR-Z]'),
+          vendor_R_T: new RegExp('node_modules/[r-tR-T]'),
+          vendor_U: new RegExp('node_modules/[uU]'),
+          vendor_V_Z: new RegExp('node_modules/[v-zV-Z]'),
```

Comment:
what's the context here? and if it's unrelated to the SG changes, can we cherry-pick it out into its own PR?

Reply:
ooooops